ch_tools/chadmin/cli/wait_group.py

```diff
@@ -109,7 +109,7 @@ def wait_replication_sync_command(
                 ctx,
                 f"SYSTEM SYNC REPLICA {full_name}",
                 format_=None,
-                timeout=timeout,
+                timeout=time_left,
                 settings={"receive_timeout": timeout},
```

Comment:
Imo it will becomes overcomplicated...
wdyt about
```suggestion
                timeout=replica_timeout.total_seconds(),
                settings={"receive_timeout": time_left},
```